MapboxDirections/MBRouteStep.swift

```diff
@@ -222,6 +222,11 @@ public enum ManeuverType: Int, CustomStringConvertible {
      */
     case passWaypoint // v4
     
+    /**
+     The step requires the user to exit a roundabout.
```

Comment:
> This step requires the user to exit a roundabout (traffic circle or rotary).
> 
> This maneuver type follows a `.takeRoundabout` or `.takeRotary` maneuver. It is only used when `RouteOptions.includeExitRoundaboutManeuver` is set to `true`.

Also add a new paragraph to the `.takeRoundabout` and `.takeRotary` documentation about this maneuver type:

> If `RouteOptions.includeExitRoundaboutManeuver` is set to `true`, this step is followed by an `.exitRoundabout` maneuver.

Reply:
For `exitRotary`:

> This step requires the user to exit a large, named roundabout (traffic circle or rotary).
> 
> This maneuver type follows a `.takeRotary` maneuver. It is only used when `RouteOptions.includeExitRoundaboutManeuver` is set to `true`.